sentences/cs/_common.yaml

```diff
@@ -7,11 +7,56 @@ responses:
     no_device_class: "{{ area }} neobsahuje {{ device_class }}"
     no_entity: "Žádné zařízení ani entita se nejmenuje {{ entity }}"
     handle_error: "Při zpracování došlo k neočekávané chybě."
-lists: {}
+lists: 
+  color:
+    values:
+      - in: "bíl(á|ý|é|ou)"
+        out: "white"
+      - in: "čern(á|ý|é|ou)"
+        out: "black"
+      - in: "červen(á|ý|é|ou)"
+        out: "red"
+      - in: "oranžov(á|ý|é|ou)"
+        out: "orange"
+      - in: "žlut(á|ý|é|ou)"
+        out: "yellow"
+      - in: "zelen(á|ý|é|ou)"
+        out: "green"
+      - in: "modr(á|ý|é|ou)"
+        out: "blue"
+      - in: "fialov(á|ý|é|ou)"
+        out: "purple"
+      - in: "hněd(á|ý|é|ou)"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "(celsius | celsia)"
+      - in: "c"
+        out: "Celsia"
+      - "(fahrenheit | fahrenheita)"
+      - in: "f"
+        out: "Fahrenheita"
 expansion_rules:
+  area: "[(na|v)] {area}"
   zapnout: "(zapni|zapnout)"
-  vypnout: "(vypni|vypnout)"
+  vypnout: "(vypni|vypnout|zastavit|zastav)"
   rozsvitit: "(rozsviť|rozsvítit)"
   zhasnout: "(zhasni|zhasnout)"
+  nastav: "(nastav|nastavit)"
+  teplota: "{temperature} [stupňů] {temperature_unit}"
```

Comment:
Better would be 
`{temperature} [stupňů] [{temperature_unit}]`
Your version allow to say "15 Celsia", but not "15 stupňů", which is quite common.
Not a critical comment, however.

Reply:
Yes, found it when testing climate :)
Corrected.